spec/views/exports/lcmb_pcr_xp_concentrations_for_custom_pooling.csv.erb_spec.rb

```diff
@@ -0,0 +1,51 @@
+# frozen_string_literal: true
+
+require 'spec_helper'
+
+RSpec.describe 'exports/lcmb_pcr_xp_concentrations_for_custom_pooling.csv.erb' do
+  has_a_working_api
+
+  let(:qc_result_options_1) { { value: 1.5, key: 'concentration', units: 'ng/ul' } }
+  let(:qc_result_options_2) { { value: 2.7, key: 'concentration', units: 'ng/ul' } }
+
+  let(:well_a1) do
+    create(:v2_well, position: { 'name' => 'A1' }, qc_results: create_list(:qc_result, 1, qc_result_options_1))
+  end
+  let(:well_b1) do
+    create(:v2_well, position: { 'name' => 'B1' }, qc_results: create_list(:qc_result, 1, qc_result_options_2))
+  end
+
+  let(:ancestor_well_a1) do
+    create(:v2_well, position: { 'name' => 'A1' }, qc_results: create_list(:qc_result, 1, qc_result_options_1))
+  end
+  let(:ancestor_well_b1) do
+    create(:v2_well, position: { 'name' => 'B1' }, qc_results: create_list(:qc_result, 1, qc_result_options_2))
+  end
+  let(:labware) { create(:v2_plate, wells: [well_a1, well_b1], pool_sizes: [1, 1]) }
+  let(:ancestor_labware) { create(:v2_plate, wells: [ancestor_well_a1, ancestor_well_b1], pool_sizes: [1, 1]) }
```

Comment:
How/where is the labware associated with the ancestor labware?

Reply:
It isn't associated.
I think we bypass that normal requirement by assigning the plate and ancestor parameters for the screen in the before section lines 35-38. i.e. the screen doesn't care, and it's the screen we are testing here.

Reply:
Could add a comment?
I'm also now wondering why the qc results are being assigned to both ancestor and current plate wells...
This was a partial cut/paste from another similar test.